mathesar_ui/src/layouts/LayoutWithHeader.svelte

```diff
@@ -1,30 +1,67 @@
 <script lang="ts">
   import AppHeader from '@mathesar/components/AppHeader.svelte';
+  import LiveDemoBanner from '@mathesar/components/LiveDemoBanner.svelte';
 
   export let fitViewport = false;
```

Comment:
We should probably change the name of this variable to better indicate it's functionality. I'd like to get some suggestions.

Reply:
What has changed about the functionality of `fitViewport`?